zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/tasks/incident/ElasticsearchIncidentUpdateRepository.java

```diff
@@ -129,6 +151,25 @@ public CompletionStage<List<String>> analyzeTreePath(final String treePath) {
             response -> response.tokens().stream().map(AnalyzeToken::token).toList(), executor);
   }
 
+  private Query createProcessInstanceDeletedQuery(final long processInstanceKey) {
+    final var stateValues =
```

Comment:
:thought_balloon:  Not really necessary or important, but it can be static (and not modifiable)

Reply:
Good catch :+1: